lms/services/lti_grading/_v13.py

```diff
@@ -107,6 +107,14 @@ def sync_grade(  # noqa: PLR0913
         taking all the necessary information as parameters.
         """
         payload = self._record_score_payload(score, user_grading_id, grade_timestamp)
+
+        if lti_registration.product_family == Family.CANVAS:
+            # By default Canvas calls to /score create a new submission
+            # Disable that behaviour and just submit the new grade.
+            # See: https://canvas.instructure.com/doc/api/score.html
+            payload["https://canvas.instructure.com/lti/submission"] = {
```

Comment:
This is a pretty bold decision for the default in the canvas API

This is the LTI standard call to record a new score but we need this parameter to avoid creating a new submission :shrug:

Reply:
Yep. Pretty dark side effect in my opinion.